templates/admin/user/edit.tmpl

```diff
@@ -177,14 +177,14 @@
 
 				<div class="inline field">
 					<div class="ui radio checkbox">
-						<input name="source" value="local" type="radio" {{if .User.UseCustomAvatar}}checked{{end}}>
-						<label>{{.locale.Tr "settings.enable_custom_avatar"}}</label>
+						<input id="source" name="source" value="local" type="radio" {{if .User.UseCustomAvatar}}checked{{end}}>
+						<label for="source">{{.locale.Tr "settings.enable_custom_avatar"}}</label>
```

Comment:
this will collide with source a few lines up.

Reply:
> this will collide with source a few lines up.

That lines are from another Auth source case. It will never be rendered at same time.

Reply:
Fixed radio ID+FOR attribute values to avoid ID collisions.